IndicoV2.Abstractions/Submissions/Models/Submission.cs

```diff
@@ -72,12 +72,14 @@ public SubmissionSs(IListSubmissions_Submissions_Submissions submission)
 
         private SubmissionStatus ConvertFromSs()
         {
-                if (!Enum.TryParse(_ssSubmission.Status.ToString().ToUpper(),out SubmissionStatus parsed))
-                {
-                    throw new NotSupportedException($"Cannot read submission status: {_ssSubmission.Status}");
-                }
+            var serializer = new SubmissionStatusSerializer();
```

Comment:
Issue is StrawberryShake serializes SubmissionStatus as `PendingReview` when it should be `PENDING_REVIEW`. I just leveraged StrawberryShake serializer to convert to expected format

Reply:
line `8274` of `indico-client-csharp/IndicoV2.StrawberryShake/Generated/IndicoGqlClient.StrawberryShake.cs`

Reply:
IMO, it would be preferrable here to use a `switch` case to map the two enums, rather than grabbing the generated serializer. Since the enum list is pretty short.  Does that make sense?

Reply:
yeah that makes sense

Reply:
added switch case